### PR TITLE
[catch2] updated catch2 to version 2.13.6

### PR DESCRIPTION
**Pull request by georg-emg**
I updated the version in the catch2 port to 2.13.6. I also replaced the old CONTROL file with a proper manifest.

### CODE REVIEW

File: ports/catch2/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "catch2",
+  "version-string": "2.13.6",
```
Comment (NancyLi1013) [inline ports/catch2/vcpkg.json:3]: This filed might be `version-semver` here. Please refer to https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md.

File: versions/c-/catch2.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "0bbe381f5f7a5df720a1bdaca43da2ec942004c1",
+      "version-string": "2.13.6",
```
Comment (NancyLi1013) [inline versions/c-/catch2.json:5]: The same as above.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]
It seems that you forgot to update portfile.cmake file.

**Comment by georg-emg** [comment]
> It seems that you forgot to update portfile.cmake file.

Only took me two tries to fix! (I hope)

File: ports/catch2/vcpkg.json (hunk 1 above)
Comment (NancyLi1013) [reply ports/catch2/vcpkg.json:3]: Seems you missed this point.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Pull request merged**